### ISSUE TITLE
Dependency security issue, npm --> ... --> mem

### ISSUE THREAD

**Issue by Vektrat** [vesln/hippie#54]
Hi, dependabot is complaining about a [security issue of moderate severity](https://github.com/sindresorhus/mem/commit/da4e4398cb27b602de3bd55f746efa9b4a31702b) in the `mem` package, ultimately used by `npm` in `hippie`

```bash
=> Found "mem@1.1.0"
info Reasons this module exists
   - "hippie#npm#lock-verify#@iarna#cli#yargs#os-locale" depends on it
```

I believe the `npm` dependency of hippie should be upgraded to https://github.com/npm/cli/releases/tag/v6.14.7

**Comment by morfie78** [comment]
Fixed in today's 0.6.1 release.  Thank you for reporting!

**Comment by Vektrat** [comment]
> Fixed in today's 0.6.1 release. Thank you for reporting!

was not sure how active this repo was, so i was going to submit a PR later on, very glad to see this was addressed so promptly, thanks ^^

**Comment by morfie78** [comment]
Sure thing.  I'm still trying to get the other dependencies updated, but I lack the access to those repos.  I've put in requests, and will hopefully get to update them soon.  (hydro, fload, etc.)
